kits/rosie/rosie_demo.cpp

```diff
@@ -239,7 +280,7 @@ int main(){
       Eigen::VectorXd masses;
       arm_model->getMasses(masses);
       Eigen::VectorXd effort(arm_dofs.size());
-      auto base_accel = fbk[base.num_wheels_ + 0].imu().accelerometer().get();
+      auto base_accel = fbk[base_num_wheels + 0].imu().accelerometer().get();
```

Comment:
Why is `+ 0` here?

Reply:
oh it was "+1" in matlab so I just left it so that I'd be able to cross reference more easily, but i guess I can take that out now